e2e/cdc/test.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+mkdir -p /output/hoodie_test.parquet
+set -e
+
+docker-compose up -d connect connect
+docker-compose up -d hive hive
+docker-compose up --exit-code-from schema-registry-register schema-registry-register
+docker-compose up --exit-code-from mysql-client mysql-client
+sleep 10
```

Comment:
Now add a wait for kafka topic with X messages (I have such a script in the kafka e2e)